.github/workflows/sonar.yml

```diff
@@ -0,0 +1,63 @@
+name: "[Code Quality Check] Sonar Scan"
+
+on:
+  workflow_dispatch:
+    inputs:
+        ref:
+          description: "Which (branch or tag) you want to perform a sonar scan ?"
+          required: false
+          default: develop
+  push:
+    branches:
+      - develop
+  schedule:
+    - cron: "9 19 * * 1-5" # every working day at 7pm we will do a daily scan on develop
```

Comment:
Do we really need both?

Reply:
you right, no we don't need both, a schedule is sufficient

Reply:
Done ☑️ thanks